Tests/test_utils.c

```diff
@@ -20,6 +22,7 @@ TEST_TEAR_DOWN(Utils) {
 }
 
 TEST_GROUP_RUNNER(Utils) {
+	RUN_TEST_CASE(Utils, Utils_QuickUint32Pow10);

```

Comment:
Use spaces instead of tab. We need to enable formatter for tests as well.

Reply:
> We need to enable formatter for tests as well.
Definitely